.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+- [ ] This has been discussed in issue #..., or if not, please tell us why
+  here.
+- [ ] A short description of what this does, why it is needed; if the
+  description becomes long, the matter should probably be discussed in an issue
+  first.
+- [ ] This PR contains test cases, if meaningful.
+- [ ] A reviewer from the core maintainer team has been assigned for this PR.
+  If you don't know who could review this, please indicate so and/or ping
+  `bnjbvr`. The list of suggested reviewers on the right can help you.
+
+Please ensure all communication adheres to the [code of
+conduct](https://github.com/CraneStation/cranelift/blob/master/CODE_OF_CONDUCT.md).
```

Comment:
Could you place this between `<!--` and `-->`. That way people dont have to manually remove it before submitting the PR.